R/draft.R

```diff
@@ -132,7 +132,40 @@ draft <- function(file,
   invisible(file)
 }
 
+#' List available R Markdown template in a package
+#'
+#' List the available template by name that can be used with [draft()] to create
+#' a new document for R Markdown from a package.
+#'
+#' @param package Package to list template from. Default to **rmarkdown**
+#' @param full_path Set to `TRUE` to get the full path to the available templates
+#'
+#' @return A character vector of templates name to use within [draft()]. If
+#'   `full_path = TRUE`, it returns the full path to the templates.
+#' @export
+#' @md
+#' @examples
+#' # List rmarkdown templates & create a draft
+#' available_templates()
+#'
+#' # List rticles templates
+#' \dontrun{
+#'   available_templates("rticles")
+#' }
```

Comment:
If the package is not installed, it won't throw an error, so it should be safe not to use `\dontrun{}`.

```suggestion
#' available_templates("rticles")
```